docs/examples/rust/secure_message_client_encrypt.rs

```diff
@@ -22,7 +22,7 @@ use std::sync::Arc;
 use std::thread;
 
 use clap::clap_app;
-use themis::keys::{KeyPair, PublicKey, SecretKey};
+use themis::keys::{KeyPair, PrivateKey, PublicKey};
```

Comment:
is the keys' order change okay?

Reply:
Yeah. The order does not matter to the compiler, but the formatting tool insists that imports are to be ordered alphabetically. (I agree with it as that makes eyeballing a bit easier.)

Reply:
got it, thanks